airflow/providers/daskexecutor/__init__.py

```diff
@@ -27,16 +27,16 @@
 
 __all__ = ["__version__"]
 
-__version__ = "1.1.0"
+__version__ = "1.2.0"
 
 try:
     from airflow import __version__ as airflow_version
 except ImportError:
     from airflow.version import version as airflow_version
 
 if packaging.version.parse(packaging.version.parse(airflow_version).base_version) < packaging.version.parse(
-    "2.5.0"
+    "2.6.0"
 ):
     raise RuntimeError(
-        f"The package `apache-airflow-providers-daskexecutor:{__version__}` needs Apache Airflow 2.5.0+"
+        f"The package `apache-airflow-providers-daskexecutor:{__version__}` needs Apache Airflow 2.6.0+"
```

Comment:
also needs to be reverted back

Reply:
Yep. Just added fixup and go through it again.